cmd/soroban-rpc/internal/test/transaction_test.go

```diff
@@ -115,7 +115,7 @@ func TestSendTransactionSucceedsWithResults(t *testing.T) {
 	assert.Nil(t, response.Error)
 	assert.Equal(t,
 		[]methods.SCVal{
-			{XDR: "AAAABAAAAAEAAAAEAAAAIJL0uHwiQNNaPB0brHQ1Wg+WexdpZMjN1H4UDoxE5pQz"},
+			{XDR: "AAAABAAAAAEAAAAGAAAAIOqfy4GuVKKfazvyk4R9P9fpo2n9HICsr+xqvVcTF+DC"},
```

Comment:
not suggesting change, gets it done, but this pattern could become difficult to triage later, i.e. without a comment saying what the unmarshal'd struct is or maybe just creating the expected xdr struct and then marshaling it to this bas64 string would provide more visibility in the test of the expected value as a baseline for where it went wrong.

Reply:
Yeah. Would be way better to parse it and compare.